### ISSUE TITLE
Option to show the requester of translation

### ISSUE THREAD

**Issue by Artemis-chan** [RitaBot-Project/RitaBot#412]
Sometime someone just reacts and and removes the reaction, it can lead to confusion when someone randomly does unnecessary translation.

I think a toggle to show the username of the person who requested translation in the footer would be nice.

**Comment by stargatecmd2009** [comment]
Yes like this would be nice. with the option to jump to original message.
![image](https://user-images.githubusercontent.com/35220804/124610370-c923a200-deb3-11eb-9e4e-6b2940e02ad1.png)

**Comment by cyberlooper** [comment]
@JShep89 - This would only be possible in embed mode

**Comment by JShep89** [comment]
> @JShep89 - This would only be possible in embed mode

yea, i know.
